test/e2e/storage/drivers/csi.go

```diff
@@ -707,3 +652,35 @@ func tryFunc(f func()) error {
 	f()
 	return err
 }
+
+func generateCleanupFunc(
```

Comment:
Nit: Maybe call this "generateDriverCleanupFunc" to be more specific.

Reply:
done